### PR TITLE
Release 2.3.5 account merge

### PR DESCRIPTION
**Pull request by arfath-gwl**

### CODE REVIEW

File: buildConfig/build-extras.gradle [hunk 1]
```diff
@@ -158,6 +159,7 @@ android {
             resValue("string", "deeplink_base_url", "${staging_deeplink_base_url}")
             resValue("string", "custom_scheme_url", "${staging_custom_scheme}")
             buildConfigField 'String', 'BASE_URL', "\"${staging_base_url}\""
+            buildConfigField 'String', 'MERGE_ACCOUNT_BASE_URL', "\"${staging_base_url}\""
```
Comment (swayangjit) [inline buildConfig/build-extras.gradle]: It should have new variable for all variants like "staging_merge_base-url".

File: src/assets/i18n/en.json [hunk 2]
```diff
@@ -567,5 +567,12 @@
   "MORE_CONTENT": "Find more books and content on {{%s}} below.",
   "SEARCH_BOOK": "Search for content (Eg: ’English Stories’)",
   "TEXTBOOK": "Textbooks",
-  "EXTRACTING_CONTENT": "Extracting content"
+  "EXTRACTING_CONTENT": "Extracting content",
+  "MERGE_ACCOUNT": "Merge Account",
+  "ACCOUNT_MERGE_CONFIRMATION_HEADING": "Merge Account",
+  "ACCOUNT_MERGE_CONFIRMATION_CONTENT": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec dictum est eu orci sollicitudin faucibus.",
```
Comment (swayangjit) [inline src/assets/i18n/en.json]: Add a valid  value for ACCOUNT_MERGE_CONFIRMATION_CONTENT like "Do you like to merge account?"

File: buildConfig/build-extras.gradle (hunk 1 above)
Comment (arfath-gwl) [reply buildConfig/build-extras.gradle]: Resolved: added keys - production_merge_account_base_url, staging_merge_account_base_url, dev_merge_account_base_url

File: src/assets/i18n/en.json (hunk 2 above)
Comment (arfath-gwl) [reply src/assets/i18n/en.json]: Resolved: Used valid message - "Would you like to merge account?"

**Review by swayangjit — APPROVED** [review]

**Pull request merged**